src/features/UserList/UserList.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import {useQuery} from "react-query";
-import {getAllUsers} from '../common/api/api';
+import {getAllUsers } from '../common/api/api';
```

Comment:
seems like an extra space here, does the linter allow it?

Reply:
@Acousticdesk He missed it, probably needs to adjust the linter.